lib/modules/dosomething/dosomething_global/dosomething_global.install

```diff
@@ -67,7 +67,7 @@ function dosomething_global_update_7003(&$sandbox) {
 /**
  * Uninstall some unused modules.
  */
-function dosomething_global_update_7004() {
+function dosomething_global_update_7005() {
```

Comment:
I'm wondering since the update numbers skip from 7002 to 7005, if it's good practice to add commented, empty update functions for 7003 and 7004. What do you think?